client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -96,13 +96,18 @@ export class CheckoutThankYouHeader extends PureComponent {
 			return translate( 'You will receive an email confirmation shortly.' );
 		}
 
+		let productName = primaryPurchase.productName;
```

Comment:
What do you think about maybe putting this block in the condition for `if ( isPlan( primaryPurchase ) ) {` since it's only needed there? Not a big deal either way.